Code/GraphMol/SmilesParse/catch_tests.cpp

```diff
@@ -2278,3 +2278,44 @@ TEST_CASE("ring bond stereochemistry in CXSMILES") {
   }
   Chirality::setUseLegacyStereoPerception(oval);
 }
+
+TEST_CASE("Github #5683", "[SMILES][bug]") {
```

Comment:
This is no longer #5683 , you could call it #5722 or just briefly describe the bug (which is a good practice anyway)

Reply:
Oh, sure, I forgot about that one.